src/services/codefixes/generateAccessors.ts

```diff
@@ -104,29 +112,51 @@ namespace ts.codefix {
         return modifierFlags;
     }
 
-    export function getAccessorConvertiblePropertyAtPosition(file: SourceFile, start: number, end: number, considerEmptySpans = true): Info | undefined {
+    export function getAccessorConvertiblePropertyAtPosition(file: SourceFile, start: number, end: number, considerEmptySpans = true): InfoOrError | undefined {
         const node = getTokenAtPosition(file, start);
         const cursorRequest = start === end && considerEmptySpans;
         const declaration = findAncestor(node.parent, isAcceptedDeclaration);
         // make sure declaration have AccessibilityModifier or Static Modifier or Readonly Modifier
         const meaning = ModifierFlags.AccessibilityModifier | ModifierFlags.Static | ModifierFlags.Readonly;
-        if (!declaration || !(nodeOverlapsWithStartEnd(declaration.name, file, start, end) || cursorRequest)
-            || !isConvertibleName(declaration.name) || (getEffectiveModifierFlags(declaration) | meaning) !== meaning) return undefined;
+
+        if (!declaration) {
+            return {
+                error: Diagnostics.Could_not_find_property_for_which_to_generate_accessor.message
+            };
+        }
+
+        if (!(nodeOverlapsWithStartEnd(declaration.name, file, start, end) || cursorRequest)) {
+            return undefined;

```

Comment:
This should just check that the span is in a sensible place to offer the refactor. Should we return the "Could not find property" message here?